src/python-frontend/python_converter.cpp

```diff
@@ -539,7 +584,8 @@ exprt python_converter::get_block(const nlohmann::json &ast_block)
     case StatementType::IF_STATEMENT:
     case StatementType::WHILE_STATEMENT:
     {
-      get_conditional_stms(element, block);
+      exprt cond = get_conditional_stm(element);
```

Comment:
Why this change? It is not clear from the title/description

Reply:
"get_conditional_stms was previously used solely for converting if/else and while statements. In this context, we passed a target block (function), and the statement was inserted as an operand. However, with ternary operators, we now have if/else as expressions.